### PR TITLE
Login overlay 2

### PR DESCRIPTION
**Pull request by tobara**

### CODE REVIEW

**Comment by kevlarr** [comment]
It seems like this PR should really be compared against the `login_overlay` branch rather than master, since you seem to have branched off of `login_overlay`... The result is that it looks like you're trying to add everything that you're already adding in the earlier PR..?

It makes it harder to review this one, since it's tough to tell what's different between this one and the other PR... Have to go commit by commit

File: public/javascripts/landing.js [hunk 1]
```diff
@@ -2,8 +2,11 @@
 function openForms(e) {
     e.preventDefault();
     formName = e.currentTarget.name;
-    tabForm(e, formName);
+    tabBtns = document.getElementsByClassName('tablinks');
+    tabForm(event, formName);
```
Comment (kevlarr) [inline public/javascripts/landing.js]: `event` should just be `e`, right?

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -29,13 +30,23 @@ def create_user
...
+  end
+
+  config.before(:each) do
+    DatabaseCleaner.start
+  end
+
+  config.after(:each) do
+    DatabaseCleaner.clean
+  end
+
```
Comment (kevlarr) [inline spec/spec_helper.rb:51]: Oh god I forgot about all of this `DatabaseCleaner` stuff.... Setting up test environments is just... ugh.

**Comment by tobara** [comment]
Correct.  I planned to merge the first, but branched off it instead.
Probably should of merged them together.   There was mostly styling changes
for the most part on the second.

**Pull request closed without merging**